config.js

```diff
@@ -119,6 +119,16 @@ config.iota = {
         port: 4061
     },
 
+    authentication: {
+        enabled: false,
+        type: 'keystone',
+        header: 'X-Auth-Token',
+        host: 'localhost',
+        port: '5000',
+        user: 'IOTA_AUTH_USER',
```

Comment:
I like the Username and Password reflecting the names of the `ENV` variables, but there really needs to be something warning about passing sensitive data in plain-text. Add a comment maybe? 

I can't see how you can use `config.js` and *not* leave the credentials in plain text. Any ideas?

Reply:
U only asked not to keep the credentials in plain text in https://github.com/telefonicaid/iotagent-json/pull/380#discussion_r253876709 ?

Reply:
@Jagatjot - yes and I like the way `IOTA_AUTH_USER` and `IOTA_AUTH_PASSWORD` is displayed in the file, but the fact remains that `config.js` is an unencrypted file and therefore if someone writes the credentials directly into their own file they are doing something stupid.

If someone wants to create an insecure installation that is still possible, but maybe we need a comment disclaimer here saying _"do this at your own risk"_

Reply:
Yes anyone can write their own credentials in config.js file therefore there must be a comment which must contain a warning.

Reply:
Fixed 51deb3c